src/services/completions.ts

```diff
@@ -1885,7 +1892,18 @@ namespace ts.Completions {
 
                 let existingName: __String | undefined;
 
-                if (isBindingElement(m) && m.propertyName) {
+                if (isSpreadAssignment(m)) {
+                    const expression = m.expression;

```

Comment:
Ok last one change needed. 
Scope out this login into another function that sets the members in map so that it can be shared for object members and jsx attributes